client/package.json

```diff
@@ -15,7 +16,7 @@
     "styled-components": "^4.1.3"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER=none react-scripts start",
```

Comment:
why?

Reply:
When you do new or unusual things, its a good idea to explain why you have done them in the description of the pull request